.github/workflows/ci-nix.yml

```diff
@@ -241,10 +241,12 @@ jobs:
         run: nix run nixpkgs#curl -- --fail-with-body -X POST --data-binary @.codecov.yml https://codecov.io/validate
 
       - name: Build and run tests with Code Coverage
-        run: nix build -L .#ci.workspaceCov
+        run: nix run "github:Mic92/nix-fast-build?rev=4376b8a33b217ee2f78ba3dcff01a3e464d13a46" -- --skip-cached --no-nom --flake .#legacyPackages.$(nix eval --raw --impure --expr builtins.currentSystem).ci.workspaceCov
 
-      - name: Ensure lcov.info exists
-        run: test -f result/lcov.info
+      # nix-fast-build will not create `result-` (- on purpuse) if the result was already build
+      # which is kind of OK - no code changes requires no codecov message.
+      # - name: Ensure lcov.info exists
+      #   run: test -f result-/lcov.info
```

Comment:
There was a reason we introduced this once, but I guess we should notice if we don't see coverage reports where there should be some …